app/models/order.rb

```diff
@@ -19,4 +19,13 @@ def claimed?
   def out_for_delivery?
     claimed? && !delivered_at
   end
+
+  def send_order_to_bakery
+    BakeryJob.perform_later(self)
+  end
+
+  def self.completed_orders
+    response = HTTParty.get("https://cakewalkers-api.herokuapp.com/bake_jobs")
+    @orders_for_delivery = response.select { |order| order["state"] == "done" }
```

Comment:
Should this be an instance var? This is a class level method so what does the instance var belong to?